### DISCUSSION TITLE
HyperHDR + WLED Tonemapping

### DISCUSSION THREAD

**Discussion by dahnken** [awawa-dev/HyperHDR#162] [Q&A]
Just a short question, because I am a little bit stuck. Is tonemapping possible while using WLED and HyperDVR? Because it seems that turning the global HDR on seems to not fix the washed out colours. Maybe I read over the specific instructions, but as I saw that software grabbers I supported, I thought it would work out in this combination. Thanks for your help!

**Comment by awawa-dev** [comment]
Hi
If you refer to HyperHDR then you must verify turning on HDR using its live video preview. Preview should change like it's described in the manual: [LINK](https://github.com/awawa-dev/HyperHDR/wiki/Quick-start#configuring-video--system-grabber) . If it doesn't have not any effect then the logs must be verified to check out if the LUT table was found correctly or it's missing (it may happen when some user custom installation was performed instead of using the installer). WLED is just a LED driver so it should render exactly what it receives although it has some settings that could affect LED strip render quality (power limit enabled, selecting white channel algorithm for sk6812 etc) but its configuration is out of scope of this project.

**Reply by dahnken** [reply]
Thanks for your quick answer.
My fault, I meant HyperHDR. The picture is send via Android grabber APK to the Pi which runs the HyperHDR image. The Pi then transforms the picture info and sends it via Wifi to the WLED instance. But as I see it, the SDR to HDR option is only usable via an USB grabber right? Maybe I am missing something here.
Edit: Just to clarify, I am using the Controller Type "wled" in "LED-Hardware"

**Reply by awawa-dev** [reply]
Yes, if you are using HyperHDR flatbuffer or protobuffer server to receive the video stream from the external application then HDR tone mapping won't work for these channels. It's only meant to correct video streams provided by USB grabbers and by internal software grabbers that support HDR10 (Windows DirectX11 software grabber).
